hwdb.d/70-mouse.hwdb

```diff
@@ -275,6 +275,7 @@ mouse:usb:v04b3p3107:name:*
 # Kensington Expert Mouse trackball
 mouse:usb:v047dp1020:*Kensington Expert Mouse*:*
  ID_INPUT_TRACKBALL=1
+ MOUSE_DPI=400@126
```

Comment:
```suggestion
 MOUSE_DPI=400@125
```

There's a 99.9% chance that they stick with a fraction of 1000Hz and the tool is off, so let's go with 125 here. Not like anything actually uses that value anyway.

Reply:
Ah ok, I thought that (126)  seemed like a strange number but I decided to trust the tool. Thanks for making the change.